rllib/algorithms/registry.py

```diff
@@ -351,8 +351,11 @@ def _get_algorithm_class(alg: str, return_config=False) -> type:
 
 
 def get_policy_class_name(policy_class: type):
-    if policy_class.__name__ in POLICIES:
-        return policy_class.__name__
+    name = policy_class.__name__
+    if name.endswith("_traced"):
+        name = name[:-7]
```

Comment:
Can you convert this to regex? I don't trust this [:-7] thing. It will result in silent error. In regex if the pattern does not match it will error out.

Reply:
good idea